doc/classes/OS.xml

```diff
@@ -800,58 +800,58 @@
 		</method>
 	</methods>
 	<members>
-		<member name="clipboard" type="String" setter="set_clipboard" getter="get_clipboard">
+		<member name="clipboard" type="String" setter="set_clipboard" getter="get_clipboard" default="&quot;an editor if it were&quot;">
```

Comment:
This one gets the clipboard of the system running `doctool` as default value, probably not what we want ;)

Reply:
Fixed.